### ISSUE TITLE
Including ellipses in expressions

### ISSUE THREAD

**Issue by bmaclach** [JacquesCarette/Drasil#1079]
In the derivation of IM1 for SSP, there are some equations that contain ellipses, like the ones shown below:

![image](https://user-images.githubusercontent.com/18634568/50358170-6ef5a800-0526-11e9-8f1e-c70e30a7e977.png)

I was unable to find a way to include an ellipsis in an Expr in Drasil. There are also some cases where there is an ellipsis between equations, like below:

![image](https://user-images.githubusercontent.com/18634568/50358281-bed46f00-0526-11e9-8a49-7f5b60502aab.png)

That one can be faked with a Sentence, but we may want to express that as an Expr as well.

**Comment by smiths** [comment]
This is also a deep problem (like #1080).  We only need it currently for the derivation, so we might choose to "fake it" for now.  Although, I am not sure how to fake it for the first example above.  The second one is an acceptable hack for now, but the first one appears incorrect in the generated documentation.

**Comment by bmaclach** [comment]
@smiths Right, I had no way to fake the first case. I simply left the ellipses out with a note to add them later.

**Comment by JacquesCarette** [comment]
The semantics of ellipsis is quite complex (though there are papers out there that try). I'd rather not go there.

Luckily, in both cases, there is a formula that expresses what's going on.  The first is a fold. The second is vector recurrence relation. That is the way the semantics should be encoded.

Then we could add some display primitives to the backed of Drasil that allows us to choose an 'exploded' view of certain concepts, so as to recover the above display, if one so wishes.

**Comment by balacij** [comment]
The ellipses in the first picture confuse me. It looks like it's the same as the second picture (just shown differently). The second one looks like it could be written with a single piecewise function (?).

**Comment by smiths** [comment]
@balacij you can see the derivation of the equation with the ellipses from [SSP](https://jacquescarette.github.io/Drasil/examples/ssp/SRS/srs/SSP_SRS.html) for IM:fctSfty.  There is a recursive relation where a variable with subscript n is defined in terms of the same variable, but with subscript n-1.  The relevant part for the first ellipse appears at the bottom of the first derivation.

I really like Jacques's idea of capturing the recurrence relation mathematically and then generating a view that is easy for the readers of the documents.
